tests/unit/validation_schemas_test.py

```
@@ -0,0 +1,292 @@
+        "household": [{"livingRenting": True}],
+        "person": [],
+        "withholdPayload": False
+    }
+    
+    # No head of household - should fail
+    test_data = base_data.copy()
+    test_data["person"] = [{"age": 25, "householdMemberType": "Child"}]
+    
+    try:
+        EligibilityRequest(**test_data)
+        assert False, "Should have failed - no head of household"
```

Ruff (0.11.9)

Do not `assert False` (`python -O` removes these calls), raise `AssertionError()`

Replace `assert False`

(B011)